Automattic/wp-calypso
# Jetpack Connect: Redirect Pressable partner plan connection after authorize

As part of activity log and rewind, we need users to approve credential collection after authorizing the connection to their new site. This PR will redirect all Pressable sites that are connecting for a partner plan to a separate authorize screen to approve the rewind connection.

To test:

- Checkout branch locally
- Run `npm start`. 
    - If you already have Calypso running, stop it and restart so that you get the updated config flags
- Create a new Pressable site
- Connect your site
- This is janky, but grab the full connection URL once you're at WP.com, before the authorization details are taken out.
- Replace `wordpress.com` with `calypso.localhost:3000` and then go to that URL 🤦‍♂️ 
- On the authorization page, ensure you see a co-branded form
- After authorizing, ensure you're redirected to proper URL (at time of this PR being created, it's redirecting to `/me`. We need the updated URL from the Activity Log folks
